Index: pytest.ini
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pytest.ini	(revision 9e90682766df40141e8d77aa401fbc578278eb6a)
+++ pytest.ini	(date 1588478378547)
@@ -1,5 +1,5 @@
 [pytest]
-addopts = -s --alluredir allure-results
+addopts = -s --alluredir allure-report
 testpaths = ./scripts
 python_files = test_*.py
 python_classes= Test_*
